packages/coze-uniapp/src/api.ts

```
@@ -0,0 +1,141 @@
+  CozeAPI as InnerCozeAPI,
+  type ClientOptions as InnerClientOptions,
+  APIError,
+} from '@coze/api';
+
+import { sharedMixins } from './mixins/shared';
+import { platformMixins } from './mixins/platform';
+
+export interface ClientOptions extends InnerClientOptions {
+  onBeforeAPICall?: (
+    options: unknown,
+  ) => ({ token?: string } | void) | Promise<{ token?: string } | void>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)